cluster/environments/azure-simple/variables.tf

```diff
@@ -27,6 +27,18 @@ variable "flux_recreate" {
   default     = ""
 }
 
+# container registry to download flux image
+variable "flux_image_repository" {
+  type = "string"
```

Comment:
Default values should be isolated to one location

Reply:
This PR provides not only implementing the default values in the module, but also extending it to override those default values in environment templates.